projects/wp-nextjs/src/components/Blocks.tsx

```diff
@@ -1,9 +1,13 @@
 import { BlocksRenderer, YoutubeLiteBlock, ImageBlock } from '@headstartwp/core/react';
 import { TwitterBlock, ImageComponent, LinkBlock } from '@headstartwp/next';
-import { FC } from 'react';
```

Comment:
**question:** why are we removing FC and therefore the return type?

Reply:
`FC` is confusing, at some point it had children but in recent versions oof React types it doesnt.

I also don't like `PropsWithChildren` bc it makes children optional and sometimes we want to it be required. So for the most part, just typing the props is enough and actually allow us to better describe the component intention

Reply:
and this one doesn't need children at all so IMO best to just type the props